orchagent/fdborch.cpp

```diff
@@ -219,7 +221,8 @@ void FdbOrch::handleSyncdFlushNotif(const sai_object_id_t& bv_id,
         for (auto itr = m_entries.begin(); itr != m_entries.end();)
         {
             auto curr = itr++;
-            if (curr->second.type != "static" && (curr->first.mac == mac || mac == flush_mac) && curr->second.is_flush_pending)
+            if (curr->second.sai_fdb_type == sai_fdb_type &&
```

Comment:
Will it work if use `curr->second.sai_fdb_type  != SAI_FDB_ENTRY_TYPE_STATIC` here? If yes we can save the extra param `sai_fdb_entry_type_t& sai_fdb_type`.

Reply:
Hi @keboliu , The correct flow is to read the flush entry type from SAI API and then proceed with removing those entries. Even though current code only flush by dynamic, when we later have flush static, correctly using the type from SAI will ensure this code is not changed.
I also removed a previous todo comment which points to read the flush attribute to infer type rather than assuming it.